### PR TITLE
Log info when pre upgrade checks

### PR DESCRIPTION
**Pull request by dstandish**
Let users know what's happening when pre-migration checks are running.

### CODE REVIEW

File: airflow/utils/db.py [hunk 1]
```diff
@@ -1282,6 +1287,7 @@ def _check_migration_errors(session: Session = NEW_SESSION) -> Iterable[str]:
         check_bad_references,
     )
     for check_fn in check_functions:
+        log.info("running pre-migration check %s", check_fn.__name__)
```
Comment (ashb) [inline airflow/utils/db.py]: This is the one I'm not sure about - if the check is a no-op (which it should be for most of the time once it's run once?) then isn't this just noise?

Comment (dstandish) [inline airflow/utils/db.py]:
```suggestion
```

Comment (dstandish) [reply airflow/utils/db.py]: ok -- removed that one @ashb

**Review by jedcunningham — APPROVED** [review]
Nit: Capitalize the first words?

File: airflow/utils/db.py [hunk 2]
```diff
@@ -1237,11 +1237,14 @@ class BadReferenceConfig:
         if "run_id" in source_table.columns:
             continue
 
+        log.info("checking for bad references in table %s", source_table.name)
```
Comment (dstandish) [inline airflow/utils/db.py]:
```suggestion
        log.info("Checking for bad references in table %s", source_table.name)
```

File: airflow/utils/db.py [hunk 3]
```diff
@@ -1237,11 +1237,14 @@ class BadReferenceConfig:
...
+
         bad_rows_subquery = bad_ref_cfg.exists_func(session, source_table, **exists_func_kwargs)
         select_list = [x.label(x.name) for x in source_table.c]
         invalid_rows_query = session.query(*select_list).filter(~bad_rows_subquery.exists())
         invalid_row_count = invalid_rows_query.count()
         if invalid_row_count <= 0:
+            log.info("no bad references in table %s", source_table.name)
```
Comment (dstandish) [inline airflow/utils/db.py]:
```suggestion
            log.info("No bad references in table %s", source_table.name)
```

File: airflow/utils/db.py [hunk 4]
```diff
@@ -1254,6 +1257,8 @@ class BadReferenceConfig:
             )
             errored = True
             continue
+
+        log.info("moving bad references from table %s", source_table.name)
```
Comment (dstandish) [inline airflow/utils/db.py]:
```suggestion
        log.info("Moving bad references from table %s", source_table.name)
```

**Comment by dstandish** [comment]
> Nit: Capitalize the first words?

done

**Review by ashb — APPROVED** [review]

**Pull request closed without merging**